dist/src/main/resources/configuration/hawkular-realm-for-dev.json

```diff
@@ -88,7 +88,7 @@
       "bearerOnly" : false,
       "publicClient": false,
       "redirectUris": ["/*", "http://localhost:8080/*", "http://127.0.0.1:8080/*"],
-      "webOrigins": ["http://localhost:2772", "http://localhost:8080", "http://127.0.0.1:8080"],
+      "webOrigins": ["http://localhost:2772", "http://localhost:8080", "http://127.0.0.1:8080", "http://localhost:9876"],
```

Comment:
Is it for automated tests? If not, then you might also want to add the IP variation to the list: 127.0.0.1:9876.